manual/index.rst

```diff
@@ -1,7 +1,7 @@
 WorldEngine - the best open-source world generator
```

Comment:
I like the change you made further down, going from "is intended to be the best open-source world generator" to "has the goal to be the best open-source world generator", it just sounds more humble. I would vote against boldness for the headline.^^